src/components/UserAccount/AccountDetail/AccountDetail.jsx

```diff
@@ -9,10 +9,10 @@ export const AccountDetail = ({ data }) => {
       <div className={styles.details}>
         <section className={styles.details__text}>
           <div>
-            Address: <CopyText value={data.address} ellipsisAfter={99} />{' '}
+            Address: <CopyText value={data.address} />{' '}
```

Comment:
Remove the prettier adding extra space at the end

Reply:
done